.github/workflows/zowe-explorer-ci.yml

```diff
@@ -52,13 +52,13 @@ jobs:
           env_vars: OS,NODE
 
       - name: Package VSIX
-        if: matrix.os == 'ubuntu-latest' # && matrix.node-version == '16.x'
+        if: matrix.os == 'ubuntu-latest' && matrix.node-version == '18.x'
         run: yarn package
         working-directory: packages/zowe-explorer
 
       - name: Archive VSIX artifact
-        if: matrix.os == 'ubuntu-latest' # && matrix.node-version == '16.x'
-        uses: actions/upload-artifact@v2
+        if: matrix.os == 'ubuntu-latest' && matrix.node-version == '18.x'
```

Comment:
On the `main` branch we still archive a VSIX artifact for Node 16. Should we update the logic there as well?

Reply:
I think we should 😋

Reply:
addressed in 9ca264f44f3581eea919903d05b6baf073e8c023